unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/AssetManager/GLTFast/Wrappers/GLTFTextureDownloaderWrapper.cs

```diff
@@ -27,7 +27,8 @@ public Texture2D GetTexture(bool linear)
         {
             Texture2D texture2D = new Texture2D(1, 1, TextureFormat.RGBA32, 0, linear);
 
-            if (asyncOp.webRequest.downloadHandler.data != null)
+            if (asyncOp.webRequest != null && asyncOp.webRequest.downloadHandler != null
```

Comment:
Its ok to do the nullchecks but i think we are not properly handling the request result here. One way of checking is: `switch (asyncOp.webRequest.result)`. We should also log `asyncOp.webRequest.error`

Reply:
But the thing is that `asyncOp.webRequest` is null. Wouldnt that generate a NRE at `asyncOp.webRequest.result` and `asyncOp.webRequest.result` ?